test/integration/cms/builtin_static_pages_test.rb

```diff
@@ -12,19 +13,5 @@ def setup
       get '/signup'
       assert_template 'layouts/main_layout'
     end
-
-    test 'builtin static page has different layout' do
-      @simple_layout = SimpleLayout.new(@provider)
-      @simple_layout.import!
-
-      @layout = @provider.layouts.create(system_name: 'custom_layout')
-      static_page = @provider.builtin_static_pages.find_by_system_name!('forum/forums/show')
-      static_page.update_attribute(:layout, @layout)
-
-      get '/forum'
-      assert_template 'layouts/custom_layout'
-    end
```

Comment:
It would be good to figure out a way to keep this test even after removing all static pages

Reply:
I tried to come up with one, but it's a bit tricky...

I am wondering whether we actually need it, because it seems that Builtin Static Pages are only used for Forum-related pages.

Reply:
I agree, I think it was meant for Forum pages.